### PR TITLE
Add server config

### PR DESCRIPTION
**Pull request by rguinea**
## What does this PR do?
- Creates a server eslint configuration 
- Updates README to add relevant changes

### CODE REVIEW

**Comment by CesarLanderos** [comment]
why did you removed the flow config on the server?

**Comment by rguinea** [comment]
Just trying to get the basic stuff on the start. Do you think it's really necessary @CesarLanderos ? Or may I add it in a future PR?

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    'import/no-extraneous-dependencies': 0,
+    'import/no-named-as-default': [0],
+    'no-confusing-arrow': 0,
+    'no-duplicate-imports': 0,
+    'no-else-return': 0,
+    'no-prototype-builtins': 0,
+    'no-unused-vars': [2, { varsIgnorePattern: '^_+$' }],
+    'no-unused-expressions': ['error', { allowTaggedTemplates: true }],
+    quotes: 0,
+    'react/jsx-filename-extension': 0,
```
Comment (charliesbox) [inline server/index.js]: these rules are just for the front end. they shouldn't be part of this config

File: browser/index.js [hunk 2]
```diff
@@ -1,44 +1,16 @@
...
-    'flowtype/boolean-style': [2, 'boolean'],
-    'flowtype/no-dupe-keys': 2,
-    'flowtype/object-type-delimiter': 2,
-    'flowtype/require-valid-file-annotation': [
-      2,
-      'always',
-      { annotationStyle: 'line' },
-    ],
-    'flowtype/union-intersection-spacing': [2, 'always'],
-    'global-require': 'off',
```
Comment (CesarLanderos) [inline browser/index.js]: this is needed for the browser only!

Comment (rguinea) [reply browser/index.js]: ufff! thanks!

**Pull request merged**